src/Orchard/UI/Resources/MetaEntry.cs

```diff
@@ -5,6 +5,35 @@
 namespace Orchard.UI.Resources {
     public class MetaEntry {
         private readonly TagBuilder _builder = new TagBuilder("meta");
+        
+        public MetaEntry()
+        {
+
+        }

```

Comment:
It is also being used in 5 references with no argument with optional,
except for the following, everything else can be changed to with parameters since they are using name, content only and they are in order and don't have to pass null for rest of them.
public static MetaEntry Combine(MetaEntry meta1, MetaEntry meta2, string contentSeparator) {
            var newMeta = new MetaEntry();
            Merge(newMeta._builder.Attributes, meta1._builder.Attributes, meta2._builder.Attributes);
            if (!String.IsNullOrEmpty(meta1.Content) && !String.IsNullOrEmpty(meta2.Content)) {
                newMeta.Content = meta1.Content + contentSeparator + meta2.Content;
            }
            return newMeta;
        }